app/src/main/java/com/connfa/receiver/NotifyReceiver.java

```diff
@@ -9,38 +9,88 @@
-        String text = intent.getStringExtra(AlarmTask.EXTRA_TEXT);
-        showNotification(context, eventId, day, text);
+    public void onReceive(final Context context, Intent intent) {
+         long eventId = intent.getLongExtra(AlarmTask.EXTRA_ID, -1);
+         long day = intent.getLongExtra(AlarmTask.EXTRA_DAY, -1);
+
+        SpeakerManager speakerManager = Model.getInstance().getSpeakerManager();
+        List<Speaker> speakerList = speakerManager.getSpeakersByEventId(eventId);
+
+        EventManager eventManager = Model.getInstance().getEventManager();
+        EventDetailsEvent event = eventManager.getEventById(eventId);
+        if (event != null) {
+            showNotification(context, event, speakerList, day);
+        }
+
     }
 
-    private void showNotification(Context context, long id, long day, String text) {
-        String title = context.getString(R.string.dont_miss_it);
+    private void showNotification(Context context, EventDetailsEvent event, List<Speaker> speakerList, long day) {
+        String title = event.getEventName();
         int icon = android.R.drawable.ic_dialog_info;
 
         Intent intent = new Intent(context, HomeActivity.class);
         intent.putExtra(EventDetailsActivity.EXTRA_EVENT_ID, id);
         intent.putExtra(EventDetailsActivity.EXTRA_DAY, day);
         PendingIntent contentIntent = PendingIntent.getActivity(context, 0, intent, PendingIntent.FLAG_UPDATE_CURRENT);
 
-        NotificationCompat.Builder builder = new NotificationCompat.Builder(context);
-        builder.setSmallIcon(icon);
-        builder.setContentTitle(title);
-        builder.setContentText(text);
-        builder.setAutoCancel(true);
-        builder.setContentIntent(contentIntent);
-        builder.setDefaults(Notification.DEFAULT_ALL);
+        NotificationCompat.Builder builder = new NotificationCompat.Builder(context)
+                .setSmallIcon(icon)
+                .setContentTitle(title)
+                .setContentText(createContentText(context, event))
+                .setAutoCancel(true)
+                .setContentIntent(contentIntent)
+                .setStyle(new NotificationCompat.BigTextStyle()
+                                  .bigText(createBigText(context, event, speakerList)))
+                .setDefaults(Notification.DEFAULT_ALL);
 
         NotificationManager manager = (NotificationManager) context.getSystemService(Context.NOTIFICATION_SERVICE);
         manager.notify(0, builder.build());
     }
+
+    private CharSequence createContentText(Context context, EventDetailsEvent event) {
+        return context.getString(R.string.start_in_5_minutes_in) + event.getPlace();
+    }
+
+    private CharSequence createBigText(Context context, EventDetailsEvent event, List<Speaker> speakerList) {
+        String speakersNames = createSpeakersNames(speakerList);
+        return context.getString(R.string.start_in_5_minutes_in) + event.getPlace()
```

Comment:
I'd rather have a separate resource with 2 string parameters here, in case we want to translate